packages/frontend/src/redux/slices/status/index.js

```diff
@@ -10,3 +16,18 @@ export const selectStatusMainLoader = createSelector(selectStatusSlice, (status)
 export const selectStatusLocalAlert = createSelector(selectStatusSlice, (status) => status.localAlert || {});
 
 export const selectStatusActionStatus = createSelector(selectStatusSlice, (status) => status.actionStatus || []);
```

Comment:
I know this isn't new to this PR but shouldn't the fallback here be a `{}`? It looks like all usages of this selector rely have property access on them.

Reply:
You're right. Thanks. Done.